scripts/us_census/pep/Population_Estimate_by_ASR/county_1980_1989.py

```diff
@@ -0,0 +1,93 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+'''
+This Python Script is
+for County and State Level Data
+1980-1989.
+'''
+import os
+import pandas as pd
+from common_functions import _input_url, _replace_age
+
+
+def county1980():
+    '''
+    This Python Script Loads csv datasets
+    from 1980-1989 on a County and State Level,
+    cleans it and create a cleaned csv
+    for both County and State.
+    '''
+    # Getting input URL from the JSON file.
+    _url = _input_url("county.json", "1980-89")
+    # Contains the final data which has been taken directly and aggregated.
+    final_df = pd.DataFrame()
+    # Contains aggregated data for age and sex.
+    df_as = pd.DataFrame()
+    # Contains aggregated data for age and race.
+    df_ar = pd.DataFrame()
+    cols=['Year','geo_ID','Race',0,1,2,3,4,5,6,7\
+                    ,8,9,10,11,12,13,14,15,16,17]
+    df = pd.read_csv(_url,skiprows=7,names=cols,low_memory=False,\
+        encoding='ISO-8859-1')
+
+    df = (df.drop(cols, axis=1).join(df[cols]))
+    df['geo_ID'] = df['geo_ID'].astype(int)
+    df['geo_ID'] = [f'{x:05}' for x in df['geo_ID']]
+    df['Race'] = df['Race'].astype(str)
+    # Replacing the names with more understandable SV headings.
+    _dict = {
```

Comment:
Similar comment to the one in the other file about _dict

Reply:
All the _dict were initially created for better readability. However I have moved them inside the replacement function itself.